pkgs/applications/misc/snowsql/linux.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  rpmextract,
+  makeWrapper,
+  openssl,
+  libxcrypt-legacy,
+  zlib,
+  version,
+  pname,
+  meta,
+}:
+stdenv.mkDerivation rec {
+  inherit pname version meta;
+  src = fetchurl {
+    url = "https://sfc-repo.snowflakecomputing.com/snowsql/bootstrap/${lib.versions.majorMinor version}/linux_x86_64/snowflake-snowsql-${version}-1.x86_64.rpm";
+    sha256 = "28a0828fea48c1686eccebb196343c635d660300ff18e36ca5b5ee3c03017611";
+  };
+
+  nativeBuildInputs = [rpmextract makeWrapper];
+
+  unpackPhase = ''
+    rpmextract $src
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin/
+    mkdir -p $out/lib/
```

`mkdir` accepts multiple directories and creates them accordingly and bash, which is the default shell for the nix phases, offers brace expansion as a shorthand
```suggestion
    mkdir -p $out/{bin,lib}
```